packages/scaffold-config/src/frameworks.ts

```diff
@@ -61,6 +61,22 @@ function getBundlerDependency (bundler: WizardBundler['type'], projectPath: stri
   }
 }
 
+export const WIZARD_MOUNT_MODULES = ['cypress/react', 'cypress/react18', 'cypress/vue', 'cypress/vue2', 'cypress/angular'] as const
+
+export type WizardMountModule = typeof WIZARD_MOUNT_MODULES[number]
+
+const mountModule = (mountModule: WizardMountModule) => async (projectRoot: string) => Promise.resolve(mountModule)
+
+const reactMountModule = async (projectPath: string) => {
+  const reactPkg = await isDependencyInstalled(dependencies.WIZARD_DEPENDENCY_REACT, projectPath)
+
+  try {
+    return semver.major(reactPkg.detectedVersion!) === 18 ? 'cypress/react18' : 'cypress/react'
+  } catch (e) {
+    return 'cypress/react'
```

Comment:
Worth dumping out the error here to `debug`?

Reply:
Refactored to not throw an error by calling `semver.valid` first